lib/drivers/medtronic600/docs/packetStructure.md

```diff
@@ -45,22 +47,58 @@ A **MiniMed Message** is broken up into [USB Packet](#usb-packet)s.
 | `0x86`    |  REQUEST_LINK_KEY_RESPONSE |
 
 ### Open Connection Request
-An **Open Connection Request** is sent from the app to the Contour Next Link 2.4 to authenticate with it, and to check whether the CNL2.4 is paired with a 600-series pump.
-An **Open Connection Request** is contained in the payload of a [MiniMed Message](#minimed-message) when the Minimed Operation is `0x10` (OPEN_CONNECTION) and the data is being sent **to** the CNL2.4.
+An **Open Connection Request** is sent from the driver to the Contour Next Link 2.4 to authenticate with it.
+An **Open Connection Request** is contained in the payload of a [MiniMed Message](#minimed-message) where the Minimed Operation is `0x10` (OPEN_CONNECTION).
+
 ![Open Connection Request](images/svg/OpenConnectionRequest.svg)
 
 | Field     | Bytes | Data Type  |   Comments   |
 |-----------|:-----:|:----------:|--------------|
 | HMAC      | 32    | Bytes      | The HMAC is generated using a proprietary algorithm from the Contour Next Link 2.4's **Model and Serial Number** string, which is obtained from the Header ASTM response from sending the initial **X** message to the CNL2.4 |
 
+### Open Connection Response
+The **Open Connection Response** is currently ignored by the driver.
+
+### Read Info Request
+A **Read Info Request** is sent from the driver to the Contour Next Link 2.4 to determine whether the CNL2.4 is paired with a 600-series pump.
+A **Read Info Request** is contained in the payload of a [MiniMed Message](#minimed-message) where the Minimed Operation is `0x14` (READ_INFO). It has no message payload.
+
+### Read Info Response
+A **Read Info Response** is contained in the payload of a [MiniMed Message](#minimed-message) where the Minimed Operation is `0x14` (READ_INFO).
+
+![Read Info Response](images/svg/ReadInfoResponse.svg)
+
+| Field     | Bytes | Data Type  |   Comments   |
+|-----------|:-----:|:----------:|--------------|
+| CNL MAC   | 8     | UInt64BE   | ZigBee MAC Address of the Contour Next Link |
```

Comment:
Good point. I thought those two were interchangeable, but as you point out, it's only IEEE 802.15.4 for this.